simplq/src/store/actionStatus/actionStatusSlice.js

```diff
@@ -17,15 +17,15 @@ import { createSlice } from '@reduxjs/toolkit';
  *
  * const actionStatus = useSelector((state) => state['deleteQueue'])
  */
-const actionStatusSlice = createSlice({
+const actionStatusSlice = (createSlice) ({
   name: 'actionStatus',
```

Comment:
We don't need brackets in function calls.

Reply:
Sure, I have changed it in my latest commit e301f52